new/PRS12916MazraetaKrestos.xml

```diff
@@ -45,18 +45,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <body>
             <listPerson>
                 <person sex="1">
-                    <persName xml:lang="gez" xml:id="n1">MazraetaKrestos</persName>
+                    <persName xml:lang="gez" xml:id="n1">መዝራዕተ᎓ ክርስቶስ</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n1"/>
+                    <persName type="alt"><roleName type="title">ʾabbā</roleName>Mazrāʿǝta Krǝstos</persName>
```

Comment:
this is not type="alt" this should go into the type="normalized" tag

Reply:
I followed the "Guidelines".

Reply:
roleName can be an extra persName element but you must have the normalized version of the fidel name for proper encoding and visualization

Reply:
Ok